flash/core/utilities/imports.py

```diff
@@ -20,6 +20,8 @@
 from typing import Callable, List, Tuple, Union
 
 import pkg_resources
```

Comment:
```suggestion

```

This is unused now :)

Reply:
I would like to take it, I think, there is a lot of interesting stuff going on here. I would surely ask, at the time I get stuck. :)

Reply:
Noted ✅and modified it, across all the files.

Reply:
@uakarsh - Looks like this was left out in the commits. I've committed this suggestion, to remove `import pkg_resources` as it's unused now. :)